exporter/collector/config.go

```diff
@@ -59,6 +59,16 @@ type MetricConfig struct {
 	// Buffer size for the channel which asynchronously calls CreateMetricDescriptor. Default
 	// is 10.
 	CreateMetricDescriptorBufferSize int `mapstructure:"create_metric_descriptor_buffer_size"`
+
+	// If true, the exporter will copy OTel's service.name, service.namespace, and
+	// service.instance.id resource attributes into the GCM timeseries metric labels. This
+	// option is recommended to avoid writing duplicate timeseries against the same monitored
+	// resource. Default is true.
+	IncludeServiceResourceAttributes bool `mapstructure:"include_service_resource_attributes"`
+
+	// If provided, resources attributes with keys matching this regex will be copied into
```

Comment:
As an alternative to regex, we could consider prefixes only.  It makes this a bit less of a foot-gun, in that users can't unintentionally match multiple prefixes (e.g. service.namespace and k8s.namespace.name with .*namespace.*), and also makes it more readable (no .* regex notation anywhere).  It may not be as powerful as regex, though.  Prometheus relabel configs have proven very powerful with regex matching.

Reply:
Talked offline, decided we are gonna go with a more structured approach with just prefix matching to being and we can add regex later if requested.